packages/plugins/parcel-transformer/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Transformer } from '@parcel/plugin';
+import {
+  findDependencies,
+  getLibraryImportSpecifiers,
+  findUsages,
+} from './utils';
+
+export default new Transformer({
+  async transform({ asset }) {
+    const code = await asset.getCode();
```

Comment:
Does this need to run on all code? Maybe we could avoid the full Babel parse with a comparatively faster regex-based check to see if it's worth parsing. Parcel uses swc for its own JSTransformer now, so we could avoid running Babel on code that doesn't need it (particularly node_modules).